src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -233,17 +234,76 @@ class ApplicationManagerImpl
   DataAccessor<ApplicationSet> applications() const OVERRIDE;
   ApplicationSharedPtr application(uint32_t app_id) const OVERRIDE;
 
+#ifdef SDL_REMOTE_CONTROL
+  ApplicationSharedPtr application(
+      const std::string& device_id,
+      const std::string& policy_app_id) const OVERRIDE;
+  AppSharedPtrs applications_by_interior_vehicle_data(
+      smart_objects::SmartObject moduleDescription) OVERRIDE;
+  uint32_t GetDeviceHandle(uint32_t connection_key) OVERRIDE;
```

Comment:
@LuxoftAKutsan Why parameter is not `const` in most of functions defined within `#ifdef SDL_REMOTE_CONTROL`?